gst/tensor_aggregator/tensor_aggregator.c

```diff
@@ -994,6 +1005,14 @@ gst_tensor_aggregator_parse_caps (GstTensorAggregator * self,
     return FALSE;
   }
 
+  /**
+   * @todo flush data
+   * Check properties to detect invalid case.
+   * Assertion when in=5 out=10 flush=20 or in=10 out=5 flush=20
+   */
+  while (self->frames_in * ++count < self->frames_out);
```

Comment:
Why not ```count = (self->frames_out + self->frame_in - 1) / self->frame_in;``` ?

Reply:
I updated this with your suggestion, thanks.